applications/gfs_init.py

```diff
@@ -0,0 +1,24 @@
+from credit.nwp import build_GFS_init
+from credit import metadata
+import yaml
+import argparse
+import xarray as xr
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-c", "--config", help="Path to config file")
+args = parser.parse_args()
+with open(args.config) as config_file:
+    config = yaml.safe_load(config_file)
+
+credit_grid = xr.open_dataset('/glade/work/cbecker/miles-credit/credit/metadata/ERA5_Lev_Info.nc')
+
+date = "202503040600"
+model_levels = [ 10,  30,  40,  50,  60,  70,  80,  90,  95, 100, 105, 110, 120, 130, 136, 137]
```

Comment:
Model levels are in the ERA5 data, and we could store them in a config file or other smaller file.